doc/versions.json

```diff
@@ -0,0 +1,19 @@
+[
+    {
+        "name": "2025.0 (latest)",
```

Comment:
Could we make this file auto-generated from the releases that we have as github tags instead? It's BTW missing some version numbers, such as 2024.7.0:
https://github.com/uxlfoundation/scikit-learn-intelex/releases